examples/resources/mondoo_integration_azure/resource.tf

```diff
@@ -0,0 +1,39 @@
+terraform {
+  required_providers {
+    mondoo = {
+      source = "mondoohq/mondoo"
+    }
+  }
+}
+
+provider "mondoo" {
+  region = "us"
+}
+
+// Create a new space
+resource "mondoo_space" "azure_space" {
+  name   = "Azure Integration w Terraform"
+  org_id = "your-org-1234567"
+}
+
+// Setup the Azure integration
+resource "mondoo_integration_azure" "azure_integration" {
+  space_id  = mondoo_space.azure_space.id
+  name      = "Azure Integration w Terraform"
+  tenant_id = "ffffffff-ffff-ffff-ffff-ffffffffffff"
+  client_id = "ffffffff-ffff-ffff-ffff-ffffffffffff"
+  scan_vms  = true
+  # subscription_whitelist = ["ffffffff-ffff-ffff-ffff-ffffffffffff", "ffffffff-ffff-ffff-ffff-ffffffffffff"]
+  # subscription_blacklist = ["ffffffff-ffff-ffff-ffff-ffffffffffff", "ffffffff-ffff-ffff-ffff-ffffffffffff"]
+  credentials = {
+    pem_file = <<EOT
```

Comment:
Not a blocker, but a super cool example would be to use Azure and Mondoo provider together so that users do not have to set this manually.